webextension/content-scripts/onboarding-modal.js

```diff
@@ -0,0 +1,118 @@
+const css = `
+.minvid__modalDialog {
+  position: fixed;
+  font-family: Arial, Helvetica, sans-serif;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background: rgba(0, 0, 0, 0.4);
+  z-index: 9999999999;
+}
+
+.minvid__modalDialog > div {
+  width: 35%;
+  min-width: 350px;
+  position: relative;
+  margin: 10% auto;
+  border-radius: 2px;
+  background: #fff;
+}
+
+.minvid__modalDialog header {
+  text-align: center;
+  padding: 20px;
+  background: #008ea4;
+  color: white;
+}
+
+.minvid__modalDialog header h2 {
+  font-weight: normal;
+  font-size: 14px;
+}
+
+.minvid__modalDialog img {
+  width: 90%;
+  height: auto;
+  margin: 15px auto;
+  padding: 10px 26px 0px;
+}
+
+.minvid__modalDialog p {
+  padding: 5px 20px 30px;
+  width: 85%;
+  margin: 0 auto;
+  font-size: 14px;
+}
+
+.minvid__close {
+  cursor: pointer;
+  color: #FFFFFF;
+  float: right;
+  font-weight: bold;
+  font-size: 14px;
+}
+
+.minvid__close:hover {
+  opacity: .7;
+}
+`;
+
+function createModal() {
+  const modal = document.createElement('div');
+  modal.className = 'minvid__modalDialog';
+
+  const contentWrapper = document.createElement('div');
+
+  const closeLink = document.createElement('a');
+  closeLink.className = 'minvid__close';
+  closeLink.id = 'minvid__close';
+  closeLink.title = 'Close';
+  closeLink.textContent = 'X';
+
+  closeLink.onclick = () => {
+    document.querySelector('.minvid__modalDialog').remove();
+  };
+
+  const h2 = document.createElement('h2');
+  h2.textContent = 'We\'ve added a little something to your videos';
+
+  const img = document.createElement('img');
+  img.src = browser.extension.getURL('/onboarding.gif');
+
+  const p = document.createElement('p');
+  p.textContent = `We\'re experimenting with a new feature which allows you to
+  pop out web videos and audio from soundcloud into an always on top view so
```

Comment:
This should probably be "SoundCloud".

Reply:
Also, does any of this get localized? I see some \*.properties files above, but didn't see any similar looking translation strings.

Reply:
That's a good question, it certainly _could_ be localized.

@johngruen @clouserw Do yall know if we should support non english languages for the shield study.

Reply:
If we have localized content everywhere else, I would.  We have enough time to localize it.  /cc @flodolo

Reply:
As far as I'm aware, we only support English for Shield studies. So, we can translate this string, but I don't expect it to be used, which is suboptimal. Can someone confirm who's targeted by these studies?

Reply:
@johngruen can speak to that